tensorflow/python/keras/activations.py

```diff
@@ -42,14 +42,18 @@
 
 @keras_export('keras.activations.softmax')
 def softmax(x, axis=-1):
-  """Softmax activation function.
+  """The softmax activation function transforms the outputs so that all values are in 
+  range (0, 1) and sum to 1. It is often used as the activation for the last layer of a 
+  classification network because the result could be interpreted as a probability 
+  distribution. To calculate the softmax, all inputs `x` become exp(x) divided by the sum 
```

Comment:
I think an equation reads better than the textual description of it here (the sentence starting with "to calculate" should just show exp(x)/ tf.reduce_sum(exp(x))